util/buildRelease/smokeTest

```diff
@@ -88,7 +88,10 @@ source $CHPL_HOME/util/cron/common.bash
 
 # Disable GMP and re2 to speed up build.
 export CHPL_GMP=none
-export CHPL_REGEXP=none
+
+if [ "${CHPL_SKIP_MAKE_MASON}" == "true" ] ; then
```

Comment:
Missed this originally, but could you change `CHPL_SKIP_MAKE_MASON` to `CHPL_SMOKE_SKIP_MAKE_MASON`?